Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:10.3-slim as installer
+FROM debian:bullseye-slim as installer
```

Comment:
This is not strictly necessary, it bumps us to Debian 11 but seems like we should bring it up to date if we're needing to change all the images.